static/js/explore.js

```diff
@@ -55,7 +55,7 @@ require([
                 $('#save-cohort-btn').prop('title','Please select at least one filter.');
             }
         }
-    });
+    }); */
 
```

Comment:
This is a problem--this is how we disable the saving button so people can't try to save with no filters selected. Was this causing a problem? It really needs to remain in place.

Reply:
So the save-cohort button was not being activated if a filter was added through a slider, or a click on a chart. Also it was not disabled after using 'Clear all filters'.  So I added this to the updateFacetsData  function after a successful call. Let it me know if this works for you.

 success: function (data) {
                    if ($('#search_def p').length>0){
                      $('#save-cohort-btn').prop('disabled','');
                      if(user_is_auth) {
                        $('#save-cohort-btn').prop('title','');
                        }
                   }
 
                   }

Reply:
That should work!